packages/agent/conf/demos/knb/auckland/oid4vci_metadata/knb_van_auckland.json

```diff
@@ -65,18 +65,18 @@
           "plaats"
         ],
         "credentialSubject": {
-          "Omschrijving": {
+          "description": {
             "value_type": "string",
             "display": [
               {
-                "name": "Omschrijving"
+                "name": "Description"
```

Comment:
what should be the default locale here? dutch or english?

Reply:
-> question overwritten